src/averageAges.js

```diff
@@ -14,12 +14,25 @@
  *
  * @return {number}
  */
+
+const isObject = (value) => typeof value === 'object' && value !== null;
+
+const getAverageAge = people => isObject(people[0])
+
+  ? people.reduce((acc, { born, died }) =>
+    acc + (died - born), 0) / people.length || 0
+
+  : people.reduce((total, value) => total + value, 0) / people.length;
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const AGE_DIVIDER = 100;
+
+  const men = people.filter(({ sex, died }) =>
+    sex === 'm' && (century
```

Comment:
It would be cool to put all the "magic values" in separate constants.

Reply:
If the sex property changes to be "male" and "female" instead of "m" or "w" you have to change it in all places in your code instead of just updating a variable so it makes sense to get rid of magic values.